packages/contracts/contracts/optimistic-ethereum/OVM/accounts/OVM_ECDSAContractAccount.sol

```diff
@@ -84,12 +84,12 @@ contract OVM_ECDSAContractAccount is iOVM_ECDSAContractAccount {
             "Transaction nonce does not match the expected nonce."
         );
 
-        // TEMPORARY: Disable gas checks for mainnet.
-        // // Need to make sure that the gas is sufficient to execute the transaction.
-        // require(
-        //    gasleft() >= SafeMath.add(transaction.gasLimit, EXECUTION_VALIDATION_GAS_OVERHEAD),
-        //    "Gas is not sufficient to execute the transaction."
-        // );
+        // Need to make sure that the gas is sufficient to execute the transaction.
+        uint256 gasLimit = (transaction.gasLimit % 10000) * 100000;
```

Comment:
Can you add a comment here that links to the relevant docs OR code within geth that demonstrates this is correct?